<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewSpur</class>
 <widget class="QDialog" name="DialogNewSpur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>功能定义</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>3</y>
     <width>320</width>
     <height>441</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOk">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>默认符号</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="LbSpurJpg">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>90</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="LbPara1">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>26</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>端点数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbTermCount">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>22</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LbPara2">
   <property name="geometry">
    <rect>
     <x>513</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>端号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EdTermNum">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>56</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>例如&quot;1~20&quot;或&quot;1,2,3..&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="LbPara3">
   <property name="geometry">
    <rect>
     <x>512</x>
     <y>94</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>子块数量</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GroupList">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>391</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>端号列表</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>端号</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="spinSpurCount">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>90</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
